app/ui/tooloptions.ui

```diff
@@ -187,6 +187,16 @@
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="QCheckBox" name="useClosedBox">
+         <property name="toolTip">
+          <string>Close Polyline path (hold Ctrl to temporarily invert)</string>
+         </property>
+         <property name="text">
+          <string>Closed</string>
```

Comment:
I think that closed might be a tad too vague, maybe we could write "Closed path" instead